src/app/components/feed/jh-pushtotalk-button.html

```diff
@@ -0,0 +1,10 @@
+<button ng-class="{'btn-danger': vm.hotkeyActive }" class="btn" ng-click="vm.click($event)" title="{{vm.titleText}}">
+  <span class="glyphicon glyphicon-bullhorn" aria-hidden="true">
+  </span>
+  <span ng-show="vm.showHotkey">
+    <kbd style="background-color:#f15a5c;line-height:16px;">
```

Comment:
This still produces de 1px jump in Firefox. Better use `line-height:1em`, which works in Chrom(e/ium) and Firefox.